src/utils/icons.ts

```diff
@@ -499,15 +499,262 @@ export function getAircraftIcon(aircraft: VatsimShortenedAircraft | VatsimPilot)
         case 'ECHO':
         case 'GLST':
             return aircraftIcons.tbm7;
+        case 'A109':
+            return aircraftIcons.h160;
+        case 'A119':
```

Comment:
Please rework this to single case, for example

```suggestion
        case 'A119':
        case 'A109':
        case 'A129':
```

etc